frontend/src/app.js

```diff
@@ -85,8 +85,8 @@ export class App extends React.Component {
             path="view/:view_id"
             element={<ElementWrapper routeElement={View} />}
           />
+        <Route path="*" element={<Navigate to="dashboard" replace />} />
```

Comment:
This takes bare URLs in the tree and redirects to dashboard.

This means if you go to a project with no path afterward, you'll automatically get the dashboard page.